src/drive/web/modules/filelist/File.jsx

```diff
@@ -148,6 +149,11 @@ const File = props => {
           formatted={isDirectory(attributes) ? undefined : formattedUpdatedAt}
         />
         <Size filesize={formattedSize} />
+        {!isMobile &&
```

Comment:
I think we should not have the isMObile condition here, and just base ourselves on the presence of additionalColumns.

This will make it easier to change our business logic regarding columns, only 1 source of truth for what columns are displayed.